src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -773,11 +773,15 @@ export function buildTooltipFactory(
     viewByAttribute: IUnwrappedAttributeHeadersWithItems,
     type: string,
     config: IChartConfig = {},
+    isDualAxis: boolean = false,
 ): ITooltipFactory {
     const { separators, stackMeasuresToPercent = false } = config;
 
     return (point: IPoint, percentageValue?: number): string => {
+        const isSecondAxis = isDualAxis ? get(point, ["series", "yAxis", "opposite"], false) : false;
+        const isDualChartWithRightAxis = isDualAxis && isSecondAxis;
```

Comment:
I think we call the same two times. Could we perhaps separate this to a function
```
function isPointOnOppositeAxis(point) {
        return get(point, ["series", "yAxis", "opposite"], false) : false;
}
```
and then just use here
`const isDualChartWithRightAxis = isDualAxis && isPointOnOppositeAxis(point);`

and also use it here: https://github.com/gooddata/gooddata-react-components/pull/916/files#diff-4307d2410a1ed87f54cab715f2f0c71bR823

(it should be sufficient to have isDualAxis check only once in the second condition)

Reply:
updated :)